src/vs/workbench/contrib/debug/browser/replFilter.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Is this a 1:1 copy from the Problems? If yes we should try to reuse code and have a shared filter class.
Though I can also look into this afterwards

fyi @sandy081